clients/include/testing_ger.hpp

```diff
@@ -194,20 +195,21 @@ void testing_ger(const Arguments& arg)
             rocblas_ger<T, CONJ>(handle, M, N, &h_alpha, dx, incx, dy, incy, dA_1, lda);
         }
 
-        gpu_time_used     = (get_time_us() - gpu_time_used) / number_hot_calls;
+        gpu_time_used = (get_time_us() - gpu_time_used) / number_hot_calls;
+
         rocblas_gflops    = ger_gflop_count<T, CONJ>(M, N) / gpu_time_used * 1e6;
-        rocblas_bandwidth = (2.0 * M * N) * sizeof(T) / gpu_time_used / 1e3;
+        rocblas_bandwidth = ger_gbyte_count<T>(M, N) / gpu_time_used * 1e6;
```

Comment:
Is this already updated in ger_batched and ger_strided_batched?

Reply:
Good catch will update them now